resources/windows/wix/properties.wxs.erb

```diff
@@ -50,6 +50,10 @@
       Id="WIXUI_EXITDIALOGOPTIONALTEXT"
       Value="Manage your first resources on this node, explore the Puppet community and get support using the shortcuts in the Documentation folder of your Start Menu." />
 
+    <Property
+      Id="WIXUI_EXITDIALOGOPTIONALCHECKBOXTEXT"
+      Value="Enable Long Path Names" />
```

Comment:
Maybe we can make this more detailed: `Disable Windows path length limit (260 character MAX_PATH limitation, requires Windows 10 1607 or later)`

Taken from https://github.com/qbittorrent/qBittorrent/blob/a2ef09466f3beb07c253e6ada2e1aa815d6c358d/dist/windows/installer.nsi#L170